plugins/corsair/fu-corsair-device.c

```diff
@@ -555,6 +536,18 @@ fu_corsair_device_init(FuCorsairDevice *device)
 
 	self->device_kind = FU_CORSAIR_DEVICE_MOUSE;
 	self->vendor_interface = CORSAIR_DEFAULT_VENDOR_INTERFACE_ID;
+	self->subdeviceId = NULL;
+	self->bp = NULL;
```

Comment:
you don't need to do this; it's done by GObject

Reply:
done